awaitwhat/utils/helpers.py

```diff
@@ -0,0 +1,15 @@
+import asyncio
+import signal
+import sys
+
+from .. import dot
+
+
+def signal_handler(signalNumber, frame):
+    tt = asyncio.all_tasks()
+    print(dot.dumps(tt))
+    sys.exit(0)
```

Comment:
What does this it actually do?
If this make the process quit, we probably don't want that 🤔